libdnf/transaction/MergedTransaction.cpp

```diff
@@ -209,24 +220,68 @@ MergedTransaction::getItems()
     }
     return items;
 }
+*/
+std::vector<std::unique_ptr<Package>> MergedTransaction::get_packages() {
+    ItemPairMap itemPairMap;
 
-static std::string
-getItemIdentifier(ItemPtr item)
-{
-    auto itemType = item->getItemType();
+    // iterate over transaction
+    for (auto & trans : transactions) {
+        //auto transItems = t->getItems();
+        // iterate over transaction items
+        for (auto & ti : trans->get_packages()) {
+            Package & pkg = *ti;
+            //auto pkg = std::dynamic_pointer_cast<Package *>(ti);
+            mergeItem(itemPairMap, &pkg);
+        }
+    }
+
+    std::vector<std::unique_ptr<Package>> result;
+    return result;
+}
+
+
+static std::string getItemIdentifier(Package * pkg) {
+    return pkg->get_name() + "." + pkg->get_arch();
+}
+
+
+static std::string getItemIdentifier(CompsGroup * grp) {
+    return grp->get_group_id();
+}
+
+
+static std::string getItemIdentifier(CompsEnvironment * env) {
+    return env->get_environment_id();
+}
+
+/*
```

Comment:
I am sorry but I don't understand why the note is here? It was part of some function but now it is only left over.